docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,36 @@
+
+# Envelope - Rule 5-43  
+
+**Rule ID:** 5-43  
+**Rule Description:** Automatic fenestration shading devices shall not be modeled in the Baseline.  
```

Comment:
Need to add whether this is a mandatory check

Reply:
resolved on line 4